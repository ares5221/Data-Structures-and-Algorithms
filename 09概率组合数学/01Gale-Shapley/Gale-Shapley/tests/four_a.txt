a: g f h e
b: f e h g
c: g h e f
d: f h e g

g: d a c b
e: d c a b
h: a b c d
f: b d c a
