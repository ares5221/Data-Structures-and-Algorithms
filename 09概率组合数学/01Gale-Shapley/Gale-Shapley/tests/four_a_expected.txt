a g
b f
c h
d e
